pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -622,6 +622,21 @@ const outputLabelConfJsonParseNoretryTemplate = `{{- define "outputLabelConfJson
 	merge_json_log false
 	replace_json_log true
   </filter>
+{{ if .Target.Syslog.AddLogSource }}
+  <filter **>
+	@type record_modifier
+	<record>
+	  kubernetes_info ${if record.has_key?('kubernetes'); record['kubernetes']; else {}; end}
```

Comment:
i realize this is a backport... but should this be empty space and not empty hash?

Reply:
this is an empty hash because this expression tries to fetch the value of `kubernetes` object